tests/PHPStan/Analyser/nsrt/bug-8886.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Bug8886;
+
+use PDO;
+use function PHPStan\Testing\assertType;
+
+function testPDOStatementGetIterator(): void {
+	if (PHP_VERSION_ID < 80000) {
```

Comment:
Instead of this IF you need a `// lint` comment at the top of the file (see other tests in the same folder for inspiration)